### PR TITLE
added alex lee

### PR DESCRIPTION
**Pull request by Vishwa0721**
- **Name of the youtuber:**
- **What the channel is about (eg. web development, design, ...)**:
- **Which section is the channel in? (if you created a section, please specify why)**:
- **Why do you consider the youtuber deserves a place in this list? *What does make it awesome?***:

- Alex Lee
- Java Programming Tutorials
- Alex Lee deserves a place in this list because his tutorials are clear and the video pace is perfect for everyone. He also teaches a wide variety of topics in Java.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -60,6 +60,13 @@ Content about: Mathematics, Programming languages \
 Featured playlists: `Learn Algebra`, `C++ Tutorial`, `C# Tutorial`, `Java Video Tutorial`, `a lot of diverse topics`. \
 <br/>
 
+[<img align="left" height="94px" width="94px" alt="Alex Lee's channel's avatar" src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRvZUQVPqQ8ipj4pv9rqVt9HCKTzrRsyYdZLcQ_7Ue8U_MV_SYT%3Ahttps%3A%2F%2Fyt3.ggpht.com%2Fa%2FAATXAJznc40hgjUtfJdS_KNadxC_6isyQON0Hw54tGJ92w%3Ds900-c-k-c0x00ffffff-no-rj&usqp=CAU"/>](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA/featured)
```
Comment (JoseDeFreitas) [inline readme.md:63]:
```suggestion
[<img align="left" height="94px" width="94px" alt="Alex Lee's channel's avatar" src="https://yt3.ggpht.com/a/AATXAJznc40hgjUtfJdS_KNadxC_6isyQON0Hw54tGJ92w=s100-c-k-c0xffffffff-no-rj-mo"/>](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA)
```
Change the link of the image. The link you put is good, but it's not the same you can copy & paste right-clicking on the image, as wrote in the [guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/contributing.md).
The channel link shouldn't have any parameter. "featured" is a parameter. I know that parameter redirects you to the main page of the channel, but it's preferable not to use any parameter to keep the same as the other YouTuebers.

File: readme.md [hunk 2]
```diff
@@ -60,6 +60,13 @@ Content about: Mathematics, Programming languages \
...
+
+[**Alex Lee**](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA/featured) \
```
Comment (JoseDeFreitas) [inline readme.md:65]:
```suggestion
[**Alex Lee**](https://www.youtube.com/channel/UC_fFL5jgoCOrwAVoM_fBYwA) \
```
Remove parameter (see comment above.)

**Review by JoseDeFreitas — CHANGES_REQUESTED** [review]
Hello @Vishwa0721!

I like the channel. Its Java tutorials are good and cover complex topics around Java.

Please, look into the changes I've commented on above.
Apart from that, and as stated in the [guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/contributing.md), new channels should be put at the bottom of the proper section. The reasons of this are: - to keep the same rules as the [awesome project](https://github.com/sindresorhus/awesome). - to avoid discussion like "why is X YouTuber about this one": new YouTuber = bottom of the proper section.
I also saw a "`.replit`" file was created. I guess it is an automatically generated file? I will delete it as it doesn't do anything in here.

Fix this so I can merge the PR. Thank you for submitting it!

**Comment by Vishwa0721** [comment]
I have made the changes you asked me too. I will make another pull request.

**Comment by Vishwa0721** [comment]
i merged the changes sorry about all the pull requests

**Review by JoseDeFreitas — APPROVED** [review]
Thank you for fixing the issues and for adding this awesome channel @Vishwa0721! 🤗

**Pull request merged**